pkg/controller/azure.go

```diff
@@ -19,6 +19,8 @@ package controller
 import (
 	"time"
 
+	"github.com/crossplane/provider-azure/pkg/controller/database/mysqlserverconfiguration"
```

Comment:
Please group this import with the other `provider-azure` imports. The general scheme we follow is:
```
<stdlib imports block>
<3rd party lib imports block>
<crossplane/crossplane-runtime imports block>
<provider-azure imports block>
```

Reply:
Similar problem was observed with the following files: 
pkg/controller/database/mysqlserverconfiguration/managed.go
pkg/controller/database/mysqlserverconfiguration/managed_test.go 
Fixed.